[2025-03-09T17:47:57.579+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-09T17:47:57.741+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_nyc_fire_incidents_dag.transform_data_task manual__2025-03-09T17:47:13.789729+00:00 [queued]>
[2025-03-09T17:47:57.749+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_nyc_fire_incidents_dag.transform_data_task manual__2025-03-09T17:47:13.789729+00:00 [queued]>
[2025-03-09T17:47:57.749+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-09T17:47:57.764+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): transform_data_task> on 2025-03-09 17:47:13.789729+00:00
[2025-03-09T17:47:57.774+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=107) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-09T17:47:57.776+0000] {standard_task_runner.py:72} INFO - Started process 109 to run task
[2025-03-09T17:47:57.777+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_nyc_fire_incidents_dag', 'transform_data_task', 'manual__2025-03-09T17:47:13.789729+00:00', '--job-id', '586', '--raw', '--subdir', 'DAGS_FOLDER/fire_incidents_dag.py', '--cfg-path', '/tmp/tmp1e819b5v']
[2025-03-09T17:47:57.778+0000] {standard_task_runner.py:105} INFO - Job 586: Subtask transform_data_task
[2025-03-09T17:47:57.815+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_nyc_fire_incidents_dag.transform_data_task manual__2025-03-09T17:47:13.789729+00:00 [running]> on host 544a43034a42
[2025-03-09T17:47:57.891+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@***.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_nyc_fire_incidents_dag' AIRFLOW_CTX_TASK_ID='transform_data_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-09T17:47:13.789729+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-09T17:47:13.789729+00:00'
[2025-03-09T17:47:57.892+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-09T17:47:57.893+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-09T17:47:57.893+0000] {logging_mixin.py:190} INFO - Current task name:transform_data_task state:running start_date:2025-03-09 17:47:57.742165+00:00
[2025-03-09T17:47:57.893+0000] {logging_mixin.py:190} INFO - Dag name:etl_nyc_fire_incidents_dag and current dag run status:running
[2025-03-09T17:47:57.893+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-09T17:47:57.894+0000] {logging_mixin.py:190} INFO - Pulling xcom!!!!!
[2025-03-09T17:47:58.014+0000] {warnings.py:112} WARNING - /opt/***/dags/pull_fire_incidents.py:48: FutureWarning: Passing literal json to 'read_json' is deprecated and will be removed in a future version. To read from a literal string, wrap it in a 'StringIO' object.
  df = pd.read_json(transform_json_data)

[2025-03-09T17:47:58.157+0000] {logging_mixin.py:190} INFO - Transforming NYC Fire Incidents Data....
[2025-03-09T17:47:58.184+0000] {logging_mixin.py:190} INFO - Transformations are Complete
[2025-03-09T17:47:58.466+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-09T17:47:58.475+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-09T17:47:58.476+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=etl_nyc_fire_incidents_dag, task_id=transform_data_task, run_id=manual__2025-03-09T17:47:13.789729+00:00, execution_date=20250309T174713, start_date=20250309T174757, end_date=20250309T174758
[2025-03-09T17:47:58.499+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-09T17:47:58.499+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-09T17:47:58.499+0000] {logging_mixin.py:190} INFO - Dag name:etl_nyc_fire_incidents_dag queued_at:2025-03-09 17:47:13.811129+00:00
[2025-03-09T17:47:58.499+0000] {logging_mixin.py:190} INFO - Task hostname:544a43034a42 operator:PythonOperator
[2025-03-09T17:47:58.512+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-09T17:47:58.531+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-09T17:47:58.533+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
